packages/@aws-cdk-testing/framework-integ/test/aws-opensearchservice/test/integ.opensearch.min.ts

```diff
@@ -7,20 +7,27 @@ class TestStack extends Stack {
   constructor(scope: Construct, id: string, props?: StackProps) {
     super(scope, id, props);
 
-    // deploy the latest opensearch domain with minimal configuration
-    const domainProps: opensearch.DomainProps = {
-      version: opensearch.EngineVersion.OPENSEARCH_2_13,
-      removalPolicy: RemovalPolicy.DESTROY,
-      capacity: {
-        multiAzWithStandbyEnabled: false,
-      },
-    };
+    const versions = [
+      opensearch.EngineVersion.OPENSEARCH_2_13,
```

Comment:
I have modified to test multiple versions. 
It tests the previous version 2.13 and the newly added version 2.15.

I considered adding all versions as well, but I thought that might be overkill, so I haven't.